<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UMO9EOT" failures="0" tests="4" name="TestNG.TestNG_Exception" time="0.010" errors="0" timestamp="2022-09-17T01:07:39 IST" skipped="2">
  <testcase classname="TestNG.TestNG_Exception" name="g" time="0.008"/>
  <system-out/>
  <testcase classname="TestNG.TestNG_Exception" name="f" time="0.001"/>
  <system-out/>
  <testcase classname="TestNG.TestNG_Exception" name="h" time="0.001">
    <skipped/>
  </testcase> <!-- h -->
  <system-out/>
  <testcase classname="TestNG.TestNG_Exception" name="e">
    <skipped/>
  </testcase> <!-- e -->
  <system-out/>
</testsuite> <!-- TestNG.TestNG_Exception -->
